Sources/SwiftInspectorVisitors/PropertySyntaxVisitor.swift

```diff
@@ -109,8 +109,8 @@ public final class PropertySyntaxVisitor: SyntaxVisitor {
 
 // MARK: - PropertyInfo
 
-public struct PropertyInfo: Hashable, CustomDebugStringConvertible {
-  public struct Modifier: Hashable, OptionSet {
+public struct PropertyInfo: Codable, Hashable, CustomDebugStringConvertible {
+  public struct Modifier: Codable, Hashable, OptionSet {
```

Comment:
excited for a future where we just store the `String` modifiers. I don't really know what benefit this type is giving us. We’ll Get There™